phpstan-baseline.neon

```diff
@@ -52,24 +52,14 @@ parameters:
 
 		-
 			message: "#^Only booleans are allowed in a negated boolean, string given\\.$#"
-			count: 2
+			count: 3
```

Comment:
> This implementation does not make much sense, why should a client send both a query id and a query?

To my understanding because of this part in the Apollo documentation:
```
    1. When the client makes a query, it will optimistically send a short (64-byte) cryptographic hash instead of the full query text.
    2. If the backend recognizes the hash, it will retrieve the full text of the query and execute it.
    3. If the backend doesn't recognize the hash, it will ask the client to send the hash and the query text so it can store them mapped together for future lookups. During this request, the backend will also fulfill the data request.
```

Nr. 3 was not possible since it gave this error. 

But maybe I'm misunderstanding something, I thought this is what we discussed yesterday in the issue (https://github.com/webonyx/graphql-php/issues/789).